about.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@700;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/1a66ebfe2a.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+    <title>About Us</title>
+</head>
+<nav>
+    <i class="fa-solid fa-bars menu-bar"></i>
+</nav>
+
+<body>
+
+    <body>
+        <nav>
+            <div class="menu-icons">
+                <i class="fa-solid fa-bars menu-bar menu-bar-desk" id="menu_bar"></i>
+                <i class="fa-solid fa-xmark btn-desk" id="btn"></i>
+            </div>
+            <div class="nav1">
+                <ul>
+                    <li class="list"><i class="fa-brands fa-facebook-f facebook"></i></li>
+                    <li><i class="fa-brands fa-twitter twitter"></i></li>
+                    <li>English</li>
+                    <li class="page">My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="nav2">
+                <div class="logo">
+                    <img src="images/supercruise logo.PNG">
+                </div>
```

Comment:
- [ Optional] In order to navigate easily to the Home page when clicking on the logo you can put the `img` tag inside an anchor tag `a` with a link to the home page.
` <div class="logo">
                 <a href="./index.html>
                    <img src="images/supercruise logo.PNG">
                   </a>
                </div>`